controllers/default_controller.py

```diff
@@ -168,10 +167,58 @@ def getChassis(host=None, user=None, password=None):
                 return "Couldn't fetch EquipmentChassis", "", 500
     else:
         handle.logout()
-        return 'Forbiden', "", 403
+        return 'Forbidden', "", 403
     handle.logout()
     return data
 
+@http_body_factory
+def getServiceProfile(host=None, user=None, password=None):
+    handle = UcsHandle(host, user, password, secure=False)
+    if handle.login():
```

Comment:
so many nested if/else, could be refined such as using guard sentences, like
if  not handle.login():
        handle.logout()
        return 'Forbidden', "", 403

dosomething